src/test/java/com/digitalasset/refapps/utils/LogFinder.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2020, Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package com.digitalasset.refapps.utils;
+
+public class LogFinder {
+  private boolean found = false;
```

Comment:
```suggestion
  private volatile boolean found = false;
```
See: https://docs.oracle.com/javase/tutorial/essential/concurrency/atomic.html

Alternatively you can use an `AtomicBoolean`.

Reply:
Good catch! Fixing...